src/main/webapp/js/adminHome.es6

```diff
@@ -16,18 +16,39 @@
  * @param {String} status
  * @returns {String} a HTML row of action result table
  */
+ /**
+* Substring used to check if join link was created
+*/
+const successStatus = 'successfully created with join link:';
+/* eslint-disable no-param-reassign */
 function createRowForResultTable(shortName, name, email, institution, isSuccess, status) {
+    const linkIndex = status.indexOf(successStatus);
+    if (linkIndex !== -1) {
+        const link = status.slice(linkIndex + successStatus.length).replace(/<br>/g, '');
+        status = status.slice(0, linkIndex + successStatus.length);
+        return `
+        <tr class="${isSuccess ? 'success' : 'danger'}">
+            <td>${encodeHtmlString(shortName)}</td>
+            <td>${encodeHtmlString(name)}</td>
+            <td>${encodeHtmlString(email)}</td>
+            <td>${encodeHtmlString(institution)}</td>
+            <td>${isSuccess ? 'Success' : 'Fail'}</td>
+            <td class="td-responsive-wrap">${status} <a href="${link}">link</a></td>
```

Comment:
Since this return block's only difference is the output of the link, please simplify it. Take the line above for an example.